README.md

```diff
@@ -33,7 +37,22 @@ The mock method takes three arguments:
 
 The mock method executes insert, then update, and finally findOne and returns the result of findOne.
 
-Each argument is optional and we have some smart behavior to handle non-existant arguments
+Each argument is optional and we have some smart behavior to handle non-existant arguments:
 - `insert`, if not specified we'll grab the first part of the `update` operator
 - `update`, if not specified we'll skip the update, if a single argument is specified, we'll use the id returned by our insert operation as the first argument (the query)
-- `find`, if not specified we'll use the id returned by our insert operation
\ No newline at end of file
+- `find`, if not specified we'll use the id returned by our insert operation
+
+**The `mockMulti` method takes four arguments:**
+
+- `insert` an array of arguments to be passed to the mockCollection's insert method
+- `update` an array of arguments to be passed to the mockCollection's update method
+- `flags` an object to be passed to the mockCollection's update method
```

Comment:
`flags` should not be passed as a separate argument, we already support passing multiple arguments to the update operation.

Instead of: `collection.mockMulti({name: 'joe'}, {$set: {age: 12}}, {multi: true})`
The developer should use this: `collection.mockMulti({name: 'joe'}, [{$set: {age: 12}}, {multi: true}])`